utils/arangoWrapper.js

```diff
@@ -1,12 +1,18 @@
 const { Database } = require('arangojs')
-
+const {
+  ARANGO_PROTOCOL: protocol = "http",
+  ARANGO_HOST: host,
+  ARANGO_PORT: port,
+  ARANGO_DB: databaseName,
+  ARANGO_USER: username,
+  ARANGO_PASSWORD: password,
+} = process.env;
```

Comment:
Object destructuring for `process.env.*` would not work, or would lead to security holes in Next.js, since `process.env` undergoes special handling in order to keep server-side secrets safe and not accidentally bundle them into client-side code. All such vars must therefore be **explicitly** referenced as `process.env.YOUR_CUSTOM_ENV_VAR`

See https://nextjs.org/docs/basic-features/environment-variables